### PR TITLE
[TDV-663] Feature/add capital deployment fee

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/components/User/Fees/components/FeeList.vue [hunk 1]
```diff
@@ -101,7 +104,7 @@
         />
       </div>
     </div>
-
+    <!-- <pre>{{ fees }}</pre> -->
```
Comment (max-nes) [inline src/components/User/Fees/components/FeeList.vue]: remove comment

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ for further information about branching and tagging conventions.
 - New feature of managing pending tasks of KYC request
 - New external details viewer of KYC request
 - New release sanity check script, run it on pre-push
+- New "Capital deployment" fee type
```
Comment (max-nes) [inline CHANGELOG.md]: should be under "Unreleased"

File: src/components/User/Fees/components/FeeList.vue (hunk 1 above)
Comment (Datikzz) [reply src/components/User/Fees/components/FeeList.vue]: removed

File: CHANGELOG.md (hunk 2 above)
Comment (Datikzz) [reply CHANGELOG.md]: moved to unreleased

**Pull request merged**